bin/generate.dart

```diff
@@ -73,6 +73,14 @@ ArgParser _generateArgParser(GenerateOptions? generateOptions) {
       help: 'Support json or keys formats',
       allowed: ['json', 'keys']);
 
+  parser.addFlag(
+    'skip-object-keys',
+    abbr: 'x',
+    defaultsTo: false,
+    callback: (bool? x) => generateOptions!.skipObjectKeys = x,
+    help: 'If true - skip keys of nested objects.',
```

Comment:
Skip unnecessary keys of nested objects.

Reply:
Done